### PR TITLE
Idr0015

### PR DESCRIPTION
**Pull request by eleanorwilliams**
updated several things in the study file for idr0015 now that we have a paper associated with the study. 
- rename the directory from idr0015-UKNOWN-taraoceans to idr0015-colin-taraoceans and updated the screen name in the bulk.yml file
- changed title to be that of the paper
- changed description to be the abstract of the paper
- added publication details
- added licensing info
- added some information about the layout of the plates
- added a note to say there is ROI and feature data - did not go into details as its complicated! 

These changes also reflected in the Web UI (see idr-next/prod44)
Note, this study was added to IDR originally without any of the usual metadata files because the metadata is in a very different format that usual.  There is no library or processed data files.

### CODE REVIEW

**Comment by joshmoore** [comment]
Changing the location of the screen files will invalid bfmemo. Re-creation needs approx 40 hours. cc: @sbesson

**Comment by joshmoore** [comment]
It might be possible to either symlink appropriately or maintain duplicates to workaround this.

**Comment by sbesson** [comment]
Discussed with @eleanorwilliams. The renaming of the folder/screen file paths is probably too much given the current timelines. Adding the reference to the publication and fixing the screen name should be sufficient first steps for the upcoming production release.

**Comment by eleanorwilliams** [comment]
@sbesson directory renaming for idr0015 has been removed

File: idr0015-UNKNOWN-taraoceans/idr0015-study.txt [hunk 1]
```diff
@@ -1,40 +1,50 @@
...
 Study Person Last Name	de Vargas				
 Study Person First Name	Colomban				
 Study Person Email	vargas@sb-roscoff.fr				
 Study Person Address	Station Biologique de Roscoff				
-Study Person Roles					
+Study Person Roles	Principal Investigator				
 					
-Term Source Name					
-Term Source File					
+Term Source Name	NCBITaxon	EFO
```
Comment (sbesson) [inline idr0015-UNKNOWN-taraoceans/idr0015-study.txt]: Missing tab?

Comment (eleanorwilliams) [reply idr0015-UNKNOWN-taraoceans/idr0015-study.txt]: The tab is there, don't know why it looks like there isn't one in github

Comment (sbesson) [reply idr0015-UNKNOWN-taraoceans/idr0015-study.txt]: You're right. GitHub diff issue

**Comment by sbesson** [comment]
Tested `stats.py` from the merge branch. At the moment, the directory names in this directory are expected to match the screen names in IDR. A workaround for now is to create a symlink as `ln -s idr0015-UNKNOWN-taraoceans/ idr0015-colin-taraoceans`.

**Comment by sbesson** [comment]
Discussed with @jburel, integrating this as (hopefully) the final metadata PR prior to the IDR 0.4.4 release

**Pull request merged**